deploy/setup.yml

```diff
@@ -0,0 +1,55 @@
+- hosts: machines
+  become: yes
+  vars:
+    main_server_public: '{{ install_dir }}/main_server/public'
+  tasks:
+
+    - name: Installing npm in main_server
+      npm:
+        path: '{{ install_dir }}/main_server'
+
+    - name: Installing npm in main_server/public/js
+      npm:
+        path: '{{ main_server_public }}/js'
+
+    - copy:
+        src: '{{ main_server_public }}/js/node_modules/jquery/dist/jquery.min.js'
+        dst: '{{ main_server_public }}/js'
+    - copy:
+        src: '{{ main_server_public }}/js/node_modules/file-saver/FileSaver.min.js'
+        dst: '{{ main_server_public }}/js'
+    - copy:
+        src: '{{ main_server_public }}/js/node_modules/materialize-css/dist/js/materialize.min.js'
+        dst: '{{ main_server_public }}/js'
+    - copy:
+        src: '{{ main_server_public }}/js/node_modules/materialize-css/dist/css/materialize.min.css'
+        dst: '{{ main_server_public }}/css'
+
+    - name: Removing the node_modules directory
+    - file:
+        state: absent
+        path: '{{ main_server_public }}/js/node_modules/'
+
+    - name: Installing npm in load_balancer
+      npm:
+        path: '{{ install_dir }}/load_balancer' 
+
+    - name: Installing npm in execution nodes
+      npm:
+        path: '{{ install_dir }}/execution_nodes/execution_node_{{ item }}'
+      with_sequence: start=1 count={{ number_of_execution_nodes }}
+
+    - name: Installing npm in util directory
+      npm:
+        path: '{{ install_dir }}/util'
+
+    - debug:
+        msg: "Creating SSL Certificates"
+
+    - name: Running keys.sh
+      shell: '{{ playbook_dir }}/keys.sh'
```

Comment:
Once the playbook is moved to plays/, the path shall be 'cd {{ playbook_dir }}/.. && bash keys.sh'